x-pack/plugins/observability/public/components/shared/exploratory_view/embeddable/embeddable.tsx

```diff
@@ -99,24 +142,107 @@ export default function Embeddable({
             />
           </EuiFlexItem>
         )}
+        {appendHeader}
+        {showExploreButton && (
+          <EuiFlexItem grow={false}>
+            <EuiButtonIcon href={href} size="s" iconType="visBarVerticalStacked" />
```

Comment:
We should use `navigateToApp` or `navigateToUrl` within an _onClick_ handler here (with the `ev.preventDefault()`). Having the `href` only causes a reload of the whole Kibana app.